php-practice/section11.php

```diff
@@ -208,10 +226,20 @@ public function dropOnIsaacNewtonsHead() {
  * Create a new Fruit object, and a new Apple object
  * Try to call dropOnIsaacNewtonsHead() on each of these objects.
  * Which will work? Which won't? Why not?
- *
- * ANSWER HERE:
+ * ANSWER HERE: The $apple one will work because the function is inside
+ * the function dropOnIsaacNewtonsHead() is inside that class. The $fruit one won't because it can't access
+ * the dropOnIsaacNewtonsHead(). And Apple is not a child of Fruit.
  */
```

Comment:
You're right, Apple is not a child of Fruit. That was my mistake. Can you fix that?

Note, even after you fix it, it will fail for the Fruit object because Fruit does not have that function.